packages/vue-components/src/panels/PanelBase.js

```diff
@@ -135,26 +135,13 @@ export default {
       }
 
       if (this.localExpanded) {
-        /*
-          Collapse panel.
-          Panel's maxHeight is 'none' at the moment, as event listener set it to 'none' after expansion.
-          Thus, we need to reset the maxHeight to its current height for collapse transition to work.
-        */
-        this.$refs.panel.style.maxHeight = `${this.$refs.panel.scrollHeight}px`;
-
-        requestAnimationFrame(() => {
-          // To enable behaviour of auto window scrolling during panel collapse
-          if (this.$el.getBoundingClientRect().top < 0) {
-            const headerHeight = jQuery('header[sticky]').height() || 0;
-            jQuery('html').animate({
```

Comment:
let's update this with an instant scroll; this was the key issue we had back then. Again the transitions are just a "cool afterthought" =P